<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameClass</class>
 <widget class="QMainWindow" name="GameClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="GetBool">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>GetQuestionBool</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>GetQuestionInt</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="QuestionBool">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="RightAnswer">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="SecondAnswer">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ThirdAnswer">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="FourthAnswer">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="QuestionInt">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="IntAnswer">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
